libraries/adaptive-expressions/src/builtinFunctions/empty.ts

```diff
@@ -37,9 +36,7 @@ export class Empty extends ComparisonEvaluator {
      */
     private static isEmpty(instance: any): boolean {
         let result: boolean;
```

Comment:
If you don't initialize `result`, it's implicitly a `boolean | undefined` type. You should either initialize it, or amend the function return type to include `| undefined`.